### PR TITLE
kevin/ENG-961

### PR DESCRIPTION
**Pull request by kcvan**
## Description
Why did you write this code?
feat(google): add google auth and remove facebook auth except for login

to be used in conjunction with backend pr
## How to Test
- [ ] go to login/signup page
- [ ] click sign/login with google
- [ ] verify login works and everything is normal

### CODE REVIEW

**Comment by woeltjen** [comment]
> * verify login works and everything is normal

One unexpected thing is that the avatar in the top-right got replaced with Google's default "V", but my own image still shows under profile:

image

I'm not sure that's worth trying to fix (or even calling a bug), but mentioning it for completeness.

**Comment by kcvan** [comment]
@woeltjen if you refresh the page does it work? we've run into this issue where firebase doesnt update unless you refresh the page

**Comment by woeltjen** [comment]
> @woeltjen if you refresh the page does it work? we've run into this issue where firebase doesnt update unless you refresh the page

Nope, the "V" stays a "V" after refresh.

**Comment by kcvan** [comment]
@woeltjen thanks for the heads up, should work now

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -192,33 +194,69 @@ class DefaultMobileHeader extends React.Component<MobileHeaderProps> {
...
+                                if (screenType < ScreenType.TABLET) {
+                                  return (
+                                    <LazyImage
+                                      className={`${!showMenu ? 'show' : 'bee-lazy-image'}`}
+                                      height="24"
+                                      width="24"
+                                      src={
+                                        data.user.profilePicUrl
+                                          ? data.user.profilePicUrl
+                                          : user.photoURL
```
Comment (tc) [inline src/components/Header/Header.tsx]: don't understand this here

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -192,33 +194,69 @@ class DefaultMobileHeader extends React.Component<MobileHeaderProps> {
...
+                                    />
+                                  );
+                                }
+
+                                return (
+                                  <LazyImage
+                                    className={`${!showMenu ? 'show' : 'bee-lazy-image'}`}
+                                    height="48"
+                                    width="48"
+                                    src={
```
Comment (tc) [inline src/components/Header/Header.tsx]: here too, nested tenary is hard to read

File: src/components/Header/Header.tsx (hunk 1 above)
Comment (kcvan) [reply src/components/Header/Header.tsx]: thanks, fixed!

File: src/components/Header/Header.tsx (hunk 2 above)
Comment (kcvan) [reply src/components/Header/Header.tsx]: fixed!

**Pull request merged**